govcd/vapp.go

```diff
@@ -772,27 +772,50 @@ func (vapp *VApp) ChangeVMName(name string) (Task, error) {
 
 }
 
-func (vapp *VApp) DeleteMetadata(key string) (Task, error) {
-	err := vapp.Refresh()
+func (vapp *VApp) GetMetadata(requestUri string) (*types.Metadata, error) {
```

Comment:
Please add an SDK-user targeted comment (documentation) about this public function.

Reply:
Fixed.